packages/api/tests/lambdas/sf-event-sqs-to-db-records/test-index.js

```diff
@@ -386,7 +386,7 @@ test.serial('Lambda sends message to DLQ when writeRecords() throws an error', a
       WaitTimeSeconds: 10,
     });
   const dlqMessage = JSON.parse(Messages[0].Body);
```

Comment:
new data has been added to the dlq message since this test was written, so to check for what it indents, change to asking for a superset of k/vs

Reply:
they could be combined, but I kept them separate thinking it would make understanding easier:
'Lambda sends message to DLQ when writeRecords() throws an error' failed: you're not writing to the DLQ
'lambda captures error' failed: you're failing to pass through your error message (correctly)

Reply:
Fair enough - there's leeway there in increased maintenance vs clarity.